core/scripts/local/DeployEMP.js

```diff
@@ -0,0 +1,100 @@
+    await collateralToken.addMember(1, deployer, { from: deployer });
+    await collateralToken.mint(deployer, toWei("100000"), { from: deployer });
+
+    // Whitelist collateral currency
+    collateralTokenWhitelist = await AddressWhitelist.at(await expiringMultiPartyCreator.collateralTokenWhitelist());
+    await collateralTokenWhitelist.addToWhitelist(collateralToken.address, { from: deployer });
+
+    // Create identifier whitelist and register the price tracking ticker with it.
+    identifierWhitelist = await IdentifierWhitelist.deployed();
+    const priceFeedIdentifier = web3.utils.utf8ToHex("BTC/USD");
+    await identifierWhitelist.addSupportedIdentifier(priceFeedIdentifier);
+
+    // Create a mockOracle and finder. Register the mockOracle with the finder.
+    mockOracle = await MockOracle.new(identifierWhitelist.address);
+    finder = await Finder.deployed();
+    const mockOracleInterfaceName = web3.utils.utf8ToHex("Oracle");
+    await finder.changeImplementationAddress(mockOracleInterfaceName, mockOracle.address);
+
+    // Grant EMP deployer the right to register new financial templates.
+    registry = await Registry.deployed();
+    await registry.addMember(RegistryRolesEnum.CONTRACT_CREATOR, expiringMultiPartyCreator.address, {
+      from: deployer
+    });
+
+    // Create a new EMP
+    const constructorParams = {
+      expirationTimestamp: (await expiringMultiPartyCreator.VALID_EXPIRATION_TIMESTAMPS(0)).toString(),
+      collateralAddress: collateralToken.address,
+      priceFeedIdentifier: priceFeedIdentifier,
+      syntheticName: "BTCUSD",
+      syntheticSymbol: "BTCUSD",
+      collateralRequirement: { rawValue: toWei("1.5") },
+      disputeBondPct: { rawValue: toWei("0.1") },
+      sponsorDisputeRewardPct: { rawValue: toWei("0.1") },
+      disputerDisputeRewardPct: { rawValue: toWei("0.1") }
+    };
+    let _emp = await expiringMultiPartyCreator.createExpiringMultiParty.call(constructorParams, { from: deployer });
+    await expiringMultiPartyCreator.createExpiringMultiParty(constructorParams, { from: deployer });
+    emp = await ExpiringMultiParty.at(_emp);
+
+    // To create new tokens or deposit new collateral, must approve EMP to spend collateral.
+    await collateralToken.approve(emp.address, toWei("1000000"), { from: deployer });
+    // To redeem tokens, must approve EMP to spend synthetic tokens.
+    syntheticToken = await Token.at(await emp.tokenCurrency());
+    await syntheticToken.approve(emp.address, toWei("100000000"), { from: deployer });
+
+    // Create one small position so that you can create new positions from the CLI
+    // (currently, it does not support creating the first position for the EMP).
+    // Collateralize this at the minimum CR allowed.
+    await emp.create({ rawValue: toWei("1.5") }, { rawValue: toWei("1") });
+
+    // Done!
+    console.log("Created a new EMP with the configuration:", constructorParams);
```

Comment:
this looks great! do you want to also console log the address of the deployed emp? For example when testing the liquidation bot you need to know the address of the emp. 

Another nit: this `console.log` could be a `console.table` very easily :)

Reply:
Added! and made console.table it does look better.

```
➜  core git:(deploy-emp) ✗ $(npm bin)/truffle exec ./scripts/local/DeployEMP.js --network test
Using network 'test'.

Created a new EMP @0xF454D26bceeb2154F5FaC7fcEf9c81b38Ed207E7 with the configuration:
┌──────────────────────────┬───────────────────────┬──────────────────────────────────────────────┐
│         (index)          │       rawValue        │                    Values                    │
├──────────────────────────┼───────────────────────┼──────────────────────────────────────────────┤
│   expirationTimestamp    │                       │                 '1585699200'                 │
│    collateralAddress     │                       │ '0xEf688f45dbde17623cDA91C3f47cFc40c87e7379' │
│   priceFeedIdentifier    │                       │              '0x4254432f555344'              │
│      syntheticName       │                       │                   'BTCUSD'                   │
│     syntheticSymbol      │                       │                   'BTCUSD'                   │
│  collateralRequirement   │ '1500000000000000000' │                                              │
│      disputeBondPct      │ '100000000000000000'  │                                              │
│ sponsorDisputeRewardPct  │ '100000000000000000'  │                                              │
│ disputerDisputeRewardPct │ '100000000000000000'  │                                              │
```

FYI you actually don't need to get the address of the deployed EMP because you can loop through all addresses in the registry that are not the governor's address. Which is what I do in my other PR "AdvanceEMP".